api/auth/auth-router.js

```diff
@@ -16,11 +25,22 @@ router.post("/register", validateRoleName, (req, res, next) => {
    */
 });
 
-
 router.post("/login", checkUsernameExists, (req, res, next) => {
+  const UserAuth = req.body;
+
+  const token = jwt.sign(
+    {
+      username: UserAuth.username,
+      user_role: UserAuth.role_name,
+    },
+    process.env.JWT_SECRET
+  );
+  res.cookies = token;
```

Comment:
asta nu trebuie de facut

Reply:
inscrierea in cookies nu se realizeaza prin res.cookies = token si punem tokenul generat ?

Reply:
Setarea in cookies se face prin res.cookie()